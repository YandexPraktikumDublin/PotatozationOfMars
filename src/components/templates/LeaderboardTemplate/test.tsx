import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { LeaderboardTemplate } from '.'

describe('<LeaderboardTemplate />', () => {
  const title = 'Test title'

  it('should renders correct <LeaderboardTemplate />', () => {
    const wrapper = shallow(<LeaderboardTemplate title={title} />)

    expect(wrapper.contains(title)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
