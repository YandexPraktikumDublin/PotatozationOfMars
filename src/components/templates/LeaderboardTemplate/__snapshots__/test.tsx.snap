// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LeaderboardTemplate /> should renders correct <LeaderboardTemplate /> 1`] = `<Fragment />`;
